src/pybind/mgr/dashboard/module.py

```diff
@@ -248,15 +247,51 @@ class Module(MgrModule, CherryPyConfig):
     PLUGIN_MANAGER.hook.register_commands()
 
     MODULE_OPTIONS = [
-        {'name': 'server_addr'},
-        {'name': 'server_port'},
-        {'name': 'jwt_token_ttl'},
-        {'name': 'password'},
-        {'name': 'url_prefix'},
-        {'name': 'username'},
-        {'name': 'key_file'},
-        {'name': 'crt_file'},
-        {'name': 'ssl'}
+        {
+            'name': 'server_addr',
+            'type': 'str',
+            'default': '::'
```

Comment:
I added an `Option` helper in `mgr_module`, so you can basically create a new option with it:
```python
from mgr_module import Option
...

MODULE_OPTIONS = [
    Option('server_addr', '::', 'str'),
    # or
    Option(name='server_port', default=8443, type='int'),
    ....
    ]
```
Currently it's only a class wrapping `dict` and it only checks the keys ('name', 'type', etc.).

Reply:
Done